### ISSUE TITLE
QT5.9.7 Gitian Builds for windows fail with following error:

### ISSUE THREAD

**Issue by cevap** [cevap/ion#85]
```
qwin10helpers.cpp:60:37: fatal error: uiviewsettingsinterop.h: No such file or directory
 #  include <uiviewsettingsinterop.h>
```

full lxc.config, installation and build logs are attached:
[build.log](https://github.com/cevap/ion/files/2683028/build.log)
[install.log](https://github.com/cevap/ion/files/2683029/install.log)

_Originally posted by @cevap in https://github.com/cevap/ion/issue_comments#issuecomment-447594422_

**Comment by ckti** [comment]
This file is part of mingw-w64-i686-dev and mingw-w64-x86_64-dev. ran into this with openssl-1.1.1 as well.  will have a look and see how i managed to fix this (may well have been a patch).

**Comment by ckti** [comment]
Nowi rememeber what it was, qt 5.9 need to be compiled under gcc 5 and above, it will not build on trusty.

**Comment by ckti** [comment]
OK i get through qt building but now havw a ?mutex? error on bitcoin compile
[ion.build.log](https://github.com/cevap/ion/files/2683631/ion.build.log)

**Comment by cevap** [comment]
fixed & closed with #105
